File: ch1-1.rb
## Cracking the Code Algorithm 1.1

def checker(string)
  checked_letters = ""
  string.each_char do |letter|
    if checked_letters.include?(letter)
      return "The string is not unique"
    else
      checked_letters += letter
    end
      "The string is unique!"
  end
end

File: ch1-1.js
// check if a string has unique characters
var checker = function(string){
	var checkedLetters = ""

	for (var i = 0; i < string.length; i++){
		var letter = string.charAt(i)
		if (checkedLetters.includes(letter)){
			return false
		}
			checkedLetters += letter
	}
	return true
}

// console.log(checker("aaa"))
// console.log(checker("abc"))
